spec/models/document_spec.rb

```diff
@@ -398,6 +399,12 @@ def initialize(params = {})
       assert_publishing_api_unpublish(document.content_id)
     end
 
+    it "sends a payload to Rummager" do
```

Comment:
Please we say "sends a delete request to Rummager" as I don't think it's actually sending a request body payload, just a DELETE request.